src/css/main.less

```diff
@@ -639,50 +639,60 @@ fieldset[disabled] .btn-info.active {
 }
 
 .app-list-controls {
-    padding-top: 8px;
-    padding-bottom: 8px;
+  padding-top: 8px;
+  padding-bottom: 8px;
 }
 
 .filter-box {
-    max-width: 330px;
-
-    .form-control {
-        border-left-width: 0;
-        border-right-width: 0;
-        border-top-width: 2px;
-        border-bottom-width: 2px;
-
-        left: -2px;
-        -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
-        box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
-        color: #f6f6f6;
-        background: transparent;
-        border-color: #5e646c;
-    }
+  max-width: 240px;
 
-    .input-group-addon {
-        background: transparent;
-        -webkit-transition: border-color ease-in-out .15s;
-        transition: border-color ease-in-out .15s;
-        border-color: #5e646c;
-        border-width: 2px;
+  .search-icon-container {
+    .icon {
+      font-size: 1.4em;
     }
+    padding: 3px 12px;
+  }
+  .form-control {
+    padding: 6px 0;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
```

Comment:
Yep, the prefix was from the previous PR (#90). This is showing up as an addition because this block needed to be re-indented (see commit [426bdec](https://github.com/mesosphere/marathon-ui/commit/426bdec9ce82e9ad0b851b5bcb9a9121384630e5)). 

I thought it made more sense to leave the prefix in for this PR, and open a new one in which we remove all prefixes. The new issue is [here](https://github.com/mesosphere/marathon/issues/1830).